switch.go

```diff
@@ -2,18 +2,18 @@ package main
 
 import "fmt"
 
-func main () {
-  // Declare the desire
-  desire := "money"
+func main() {
+	// Declare the desire
+	desire := "money"
 
-  switch (desire) {
-    case "love":
-      fmt.Println("❤️❤️❤️❤️❤️❤️❤️")
+	switch desire {
+	case "love":
+		fmt.Println("❤️❤️❤️❤️❤️❤️❤️")
 
-    case "money":
-      fmt.Println("💰💰💰💰💰💰💰")
+	case "money":
+		fmt.Println("💰💰💰💰💰💰💰")
 
-    default:
-      fmt.Println("I cannot give you what you desire")
-  }
+	default:
+		fmt.Println("I cannot give you what you desire")
+	}
```

Comment:
Also, I feel the whitespacing is too much

Reply:
Go formatting